jboss-kie-kieserver/added/launch/jboss-kie-cli-datasource-patch.sh

```diff
@@ -1,77 +0,0 @@
-#!/usr/bin/env bash
-# Note that, this is a temporary script until https://issues.jboss.org/browse/CLOUD-2903
```

Comment:
https://issues.redhat.com/browse/CLOUD-2903 is still not resolved.
So what is the situation?

Reply:
it is half resolved, on the branch we use, it is there: https://github.com/jboss-container-images/jboss-eap-modules/pull/209
the issue stills open to get this change on master I suppose.

Reply:
ok, thanks